tap_github/streams.py

```diff
@@ -266,6 +267,34 @@ class CommunityProfileStream(GitHubStream):
     ).to_dict()
 
 
+class LanguagesStream(GitHubStream):
+    name = "languages"
+    path = "/repos/{org}/{repo}/languages"
+    primary_keys = ["repo", "org", "language_name"]
+    parent_stream_type = RepositoryStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org"]
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        """Parse the language response and reformat to return as an iterator of [{language_name: Python, bytes: 23}]."""
+        if response.status_code in self.tolerated_http_errors:
+            return []
+
+        languages_json = response.json()
+        for key, value in languages_json.items()
```

Comment:
I noticed the CI failure comes from this line. I think it just needs a colon.

```suggestion
        for key, value in languages_json.items():
```